github/resource_github_branch_protection.go

```diff
@@ -311,6 +323,12 @@ func resourceGithubBranchProtectionRead(d *schema.ResourceData, meta interface{}
 		log.Printf("[DEBUG] Problem setting '%s' in %s %s branch protection (%s)", PROTECTION_RESTRICTS_PUSHES, protection.Repository.Name, protection.Pattern, d.Id())
 	}
 
+	forcePushBypassers := setForcePushBypassers(protection, data, meta)
+	err = d.Set(PROTECTION_FORCE_PUSHES_BYPASSERS, forcePushBypassers)
+	if err != nil {
+		log.Printf("[DEBUG] Problem setting '%s' in %s %s branch protection (%s)", PROTECTION_FORCE_PUSHES_BYPASSERS, protection.Repository.Name, protection.Pattern, d.Id())
```

Comment:
Happy to change this into a `return`, but this is currently the prevailing code style here. If you expand to see the surrounding code, it's all done this way. Are you wanting this to be changed for all, or perhaps just file an issue to address this entire code at some other point?

Reply:
Argh, I can't argue with following the prevailing pattern. Ideally we'd switch this pattern all at once sometime, though it could have unintended side effects.